optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/public/kie.less

```diff
@@ -35,6 +35,13 @@
   padding-left: 0;
 }
 
+.kie-tablelist {
+  /* Firefox has a bug that cause border-collapse to not work, none of the solutions
+  in https://stackoverflow.com/questions/7517127/borders-not-shown-in-firefox-with-border-collapse-on-table-position-relative-o
+  worked. Bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=688556 */
+  border-collapse: separate;
+}
```

Comment:
Interesting. `border-collapse: collapse` seems to work in Firefox on this example: https://www.w3schools.com/CSSref/tryit.asp?filename=trycss_border-collapse. However, when I try it in OptaShift, the border sometimes isn't rendered when mouse moving over the cells. +1 for this workaround.